src/plugins/dnf/subscription_manager.py

```diff
@@ -70,6 +72,12 @@
 """
 )
 
+release_lock_waring = _(
```

Comment:
typo here (and where used):
```suggestion
release_lock_warning = _(
```